docker/orcpub/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM demonsthere/orcpub:bootstrap-lein as builder
 MAINTAINER daemonsthere@gmail.com
 
-ENV REPO="Demonsthere"
-ENV BRANCH="master"
+ARG REPO="Orcpub"
+ARG BRANCH="master"
```

Comment:
Was using master here an intentional prod vs develop mechanism? In the docker compose above, you are setting the branch to develop.